core/state/statedb.libevm.go

```diff
@@ -24,6 +24,11 @@ import (
 	"github.com/ava-labs/libevm/libevm/stateconf"
 )
 
+// TxHash returns the current transaction hash set by SetTxContext.
```

Comment:
```suggestion
// TxHash returns the current transaction hash set by [StateDB.SetTxContext].
```

This will automatically link the documentation both in IDEs and on pkg.go.dev.